README.md

```diff
@@ -43,11 +43,11 @@ On release, automated continuous integration tests run the pipeline on a full-si
     1. [`Eggnog`](https://github.com/eggnogdb/eggnog-mapper) -> Reformat TSV output "eggnog table"
     2. [`KOfamscan`](https://github.com/takaram/kofam_scan)
     3. [`HMMERsearch`](https://www.ebi.ac.uk/Tools/hmmer/search/hmmsearch) -> Ranking orfs based on HMMprofile with [`Hmmrank`](https://github.com/erikrikarddaniel/hmmrank)
-14. Taxonomical annotation:
+11. Taxonomical annotation:
     1. [`EUKulele`](https://github.com/AlexanderLabWHOI/EUKulele) -> Reformat TSV output "Reformat_tax.R"
     2. [`CAT`](https://github.com/dutilh/CAT)
     3. Choice of functional annotation: 1. [`Eggnog-mapper`](http://eggnog-mapper.embl.de) 2. [`Run-DBcan`](https://github.com/linnabrown/run_dbcan) 3. [`Hmmsearch`](https://www.ebi.ac.uk/Tools/hmmer/search/hmmsearch). Besides searching the ORFs, each ORF's hits will be ranked.
-    10 Summary statistics table. Collect_stats.R
+       10 Summary statistics table. Collect_stats.R
```

Comment:
This is not correct, right? First, there should be a `.` behind the number, it's not number 10 and should it really be indented?

Reply:
yes, this should be number 12

Reply:
And then indentation, is that correct?